articles.tex

```diff
@@ -1,212 +1,309 @@
 \part{Articles}
 
-\article{Chapter Summary}
+\chapter{Chapter Summary}
+\label{cha:chapter-summary}
 
 \section{Name}
+\label{sec:name}
 
-The name of the Fraternity chapter shall be the Eta Tau chapter of Beta Theta Pi.
+The name of the Fraternity chapter shall be the Eta Tau chapter of Beta Theta
+Pi.
```

Comment:
Pls no hard wrap at 80. Just put each sentence on its own line. Hard wrap is annoying cause if the previous line gets shortened, it doesn't unwrap